src/docs/product/alerts-notifications/alert-subscription.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Alert Rules & Settings
```

Comment:
We should probably add a section for actual alert settings (see https://sentry.io/settings/sentry/projects/javascript/alerts/)